README.md

```diff
@@ -5,6 +5,16 @@ A framework for integrating [UID2](https://github.com/IABTechLab/uid2docs) into
 
 [![License: Apache](https://img.shields.io/badge/License-Apache-green.svg)](https://www.apache.org/licenses/)
 
+## Integration Guides
+
+For the latest instructions on how to use the sdk and plugins in this repo, please read the following
```

Comment:
SDK should be all caps, no please, add ending punctuation. Suggest:
For the latest instructions on how to use the SDK and plugins in this repository, refer to the following:

Reply:
thanks i have fixed it as you suggested